src/templates/index.njk

```diff
@@ -18,7 +19,7 @@
         <article class="col-sm">
             <h2 class="h4">Book a table</h2>
             <p>Call
-                <a class="text-decoration-none" href="tel:+14153214567">+1 415-321-4567</a>
+                <a class="text-decoration-none text-dark" href="tel:+14153214567">+1 415-321-4567</a>
```

Comment:
Je to sice podle návrhu, ale takhle uživatel nebude vědět, že to je odkaz.
V návrhu to vypadá, že nikdo nepočítal s tím, že to odkaz bude.
Takže to nech jako standardní odkaz.